node-db/lesson1.md

```diff
@@ -135,6 +135,42 @@ At the endo of the wizard, you should see a new file called `package.json` in yo
 
 Here is an example `package.json` file for a project called [Passport](https://github.com/jaredhanson/passport/blob/master/package.json).
 
+# What is JSON?
```

Comment:
this is a better explanation but I think it doesn't need to come that early .. the location where it was fits better imo